subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -230,6 +236,25 @@ struct ll_conn_iso_stream *ll_conn_iso_stream_get_by_group(struct ll_conn_iso_gr
 	return NULL;
 }
 
+uint8_t ll_conn_iso_accept_timeout_get(uint16_t *timeout)
+{
+	*timeout = conn_accept_timeout;
+
+	return 0;
+}
+
+uint8_t ll_conn_iso_accept_timeout_set(uint16_t timeout)
+{
+	if ((timeout < CONN_ACCEPT_TIMEOUT_MIN) ||
+	    (timeout > CONN_ACCEPT_TIMEOUT_MAX)) {
```

Comment:
IIRC limits are inclusive, i.e. CONN_ACCEPT_TIMEOUT_MIN/MAX are both acceptable values
```suggestion
	if (!IN_RANGE(timeout, CONN_ACCEPT_TIMEOUT_MIN,
	                       CONN_ACCEPT_TIMEOUT_MAX)) {
```